src/closes-exit-int-min.c

```diff
@@ -0,0 +1 @@
+int main() { return 1 << 31; }
```

Comment:
It goes without saying that this function will never return more than `INT_MAX`.

Reply:
I just want to return a uint greater than INT_MAX or in other words a negative int here.